tests/io/mock_data.py

```diff
@@ -147,7 +147,6 @@ def __init__(self):
 
         self.signal = np.transpose(self.signal)
 
-    @contextmanager
```

Comment:
Removed the `contextmanager`s for load in this file. I think that's what we should do, but just checking in on this. Is this fine?

Reply:
It should be fine. I'll modify the mock files.